### PR TITLE
update latent walk example

### PR DESCRIPTION
**Pull request by divyashreepathihalli**

### CODE REVIEW

File: examples/generative/md/random_walks_with_stable_diffusion.md [hunk 1]
```diff
@@ -351,22 +362,28 @@ plot_grid(images, "4-way-interpolation.jpg", interpolation_steps)
...
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 210ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 205ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 11s 210ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 210ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 208ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 205ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 210ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 11s 210ms/step
+
+<ipython-input-7-932c0235506a>:51: MatplotlibDeprecationWarning: Auto-removal of overlapping axes is deprecated since 3.6 and will be removed two minor releases later; explicitly call ax.remove() as needed.
```
Comment (fchollet) [inline examples/generative/md/random_walks_with_stable_diffusion.md:378]: Please fix.

File: examples/generative/md/random_walks_with_stable_diffusion.md [hunk 2]
```diff
@@ -383,22 +400,27 @@ plot_grid(images, "4-way-interpolation-varying-noise.jpg", interpolation_steps)
...
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 11s 214ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 210ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 208ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 11s 210ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 209ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 208ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 10s 205ms/step
+ 50/50 ━━━━━━━━━━━━━━━━━━━━ 11s 213ms/step
+
+<ipython-input-7-932c0235506a>:51: MatplotlibDeprecationWarning: Auto-removal of overlapping axes is deprecated since 3.6 and will be removed two minor releases later; explicitly call ax.remove() as needed.
```
Comment (fchollet) [inline examples/generative/md/random_walks_with_stable_diffusion.md:416]: Here as well.

**Review by fchollet — COMMENTED** [review]
Nice work, thanks for the PR!

File: examples/generative/md/random_walks_with_stable_diffusion.md (hunk 2 above)
Comment (divyashreepathihalli) [reply examples/generative/md/random_walks_with_stable_diffusion.md]: done

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**